### PR TITLE
feat: Add fixtures only for new DBs, add flag to force fixture ingestion

### LINKED ISSUE #4198: [onboarding] one-time scaffolding of "demo" project on first time boot
For the use of app.phoenix.arize.com, we want the ability to ingest traces and eval annotations on first-time boot. Hook into the server boot process, detect first time launch, and ingest this data into a named project.

### PR DESCRIPTION
**Pull request by fjcasti1**
Closes #4198, #4248 

fix: Fix main.py boolean flags

### CODE REVIEW

File: src/phoenix/server/app.py [hunk 1]
```diff
@@ -111,6 +114,7 @@
 router = APIRouter(include_in_schema=False)
 
 templates = Jinja2Templates(directory=SERVER_DIR / "templates")
+NEW_DB_AGE_THRESHOLD_MINUTES = 1
```
Comment (mikeldking) [inline src/phoenix/server/app.py:117]: can you add a docstring above this? Might require some explanation

File: src/phoenix/server/app.py [hunk 2]
```diff
@@ -250,9 +258,23 @@ async def __aexit__(self, *args: Any, **kwargs: Any) -> None:
...
+            if created_at is None:
+                is_new_db = True
+            else:
+                is_new_db = datetime.now(timezone.utc) - created_at < timedelta(
+                    minutes=NEW_DB_AGE_THRESHOLD_MINUTES
+                )
+        if self._force_fixture_ingestion or is_new_db:
+            logger.info("Loading Trace Fixtures.")
+            await self._handle_tracing_fixtures()
+            logger.info("Finished loading fixtures. You may have to refresh.")
```
Comment (mikeldking) [inline src/phoenix/server/app.py:274]:
```suggestion
            logger.info("Finished loading fixtures.")
```

File: src/phoenix/server/app.py [hunk 3]
```diff
@@ -250,38 +267,74 @@ async def __aexit__(self, *args: Any, **kwargs: Any) -> None:
...
-                ),
-                get_evals_from_fixture(fixture.name),
-            )
-            project_name = fixture.project_name or fixture.name
-            logger.info(f"Loading '{project_name}' fixtures...")
-            for span in fixture_spans:
-                await self._queue_span(span, project_name)
-            for evaluation in fixture_evals:
-                await self._queue_evaluation(evaluation)
+        is_new_db = created_at is None or (
```
Comment (RogerHYang) [inline src/phoenix/server/app.py:290]: if `default` is missing, e.g. someone manually deleted it, then it's probably not new?
```suggestion
        is_new_db = created_at and (
```

File: src/phoenix/server/app.py [hunk 4]
```diff
@@ -250,38 +267,74 @@ async def __aexit__(self, *args: Any, **kwargs: Any) -> None:
...
+
+    async def _handle_tracing_fixtures(self) -> None:
+        """
+        Main handler for processing trace fixtures. Process each fixture by
+        loading its trace dataframe, gettting and processings its
+        spans and evals, and queuing.
+        """
+        try:
+            for fixture in self._tracing_fixtures:
+                trace_ds = await asyncio.get_running_loop().run_in_executor(
```
Comment (RogerHYang) [inline src/phoenix/server/app.py:304]: nit
```suggestion
            loop = asyncio.get_running_loop()
            for fixture in self._tracing_fixtures:
                trace_ds = await loop.run_in_executor(
```

File: src/phoenix/server/app.py [hunk 5]
```diff
@@ -250,38 +267,74 @@ async def __aexit__(self, *args: Any, **kwargs: Any) -> None:
...
+        )
+        return is_new_db
+
+    async def _handle_tracing_fixtures(self) -> None:
+        """
+        Main handler for processing trace fixtures. Process each fixture by
+        loading its trace dataframe, gettting and processings its
+        spans and evals, and queuing.
+        """
+        try:
```
Comment (RogerHYang) [inline src/phoenix/server/app.py:302]: maybe should `try` inside `for`?

**Review by RogerHYang — APPROVED** [review]

File: src/phoenix/server/app.py (hunk 3 above)
Comment (fjcasti1) [reply src/phoenix/server/app.py:290]: Is not deletable AFAIK

File: src/phoenix/server/main.py [hunk 6]
```diff
@@ -136,7 +136,7 @@ def _get_pid_file() -> Path:
     parser.add_argument("--export_path")
     parser.add_argument("--host", type=str, required=False)
     parser.add_argument("--port", type=int, required=False)
-    parser.add_argument("--read-only", type=bool, default=False)
+    parser.add_argument("--read-only", action="store_true", required=False)  # Default is False
```
[the remark this answers is not in the document]
Comment (fjcasti1) [reply src/phoenix/server/main.py:139]: The demo deployment is being updated in a different PR, the current demo deployment is using a previous image

File: src/phoenix/server/app.py (hunk 5 above)
Comment (fjcasti1) [reply src/phoenix/server/app.py]: Done, you were correct

**Pull request merged**